source/retryable-writes/retryable-writes.rst

```diff
@@ -519,23 +520,15 @@ The above rules are implemented in the following pseudo-code:
       if (timeoutMS == null) {
```

Comment:
I have not merged the conditions to keep the reference code straightforward with the comments.